packages/flutter/test/material/bottom_sheet_test.dart

```diff
@@ -23,6 +23,26 @@ void main() {
     expect(dyDelta1, isNot(moreOrLessEquals(dyDelta2, epsilon: 0.1)));
   }
 
+  testWidgets('Do not crash when using BottomSheet directly', (WidgetTester tester) async {
+    // Regression test for https://github.com/flutter/flutter/issues/89168
+    await tester.pumpWidget(
+      MaterialApp(
+        home: BottomSheet(
```

Comment:
For this case, an assertion seems to be more breaking, right?

Reply:
Make sense, let's have a try. Hope it won't break things. This means that we do not support using this widget alone now(updating docs?). In fact, the function is also abnormal when using directly.

Reply:
I think it is ok to use alone, you just need to provide an animation controller in that case.  There is a static method fo creating one, `BottomSheet.createAnimationController`. The assertion could provide an error message that says something like that.